svgdigitizer/electrochemistry/cv.py

```diff
@@ -312,6 +317,114 @@ def x_label(self):
             match[0], match[1], match[2] or "unknown"
         )
 
+    @property
+    @cache
+    def figure_label(self):
+        r"""
+        Return the figure name.
+
+        The figure name is read from a ``<text>`` in the SVG file
+        such as ``<text>figure: 2b</text>``.
+
+        Besides `figure`, also `fig` is acceptable in the text field.
```

Comment:
```suggestion
        Besides `figure`, also `fig` is acceptable in the text field.
```

I don't think we need that amount of convenience here. From the [Zen of Python](https://en.wikipedia.org/wiki/Zen_of_Python):

> There should be one– and preferably only one –obvious way to do it.

Reply:
see below comment on `scan rate`